setup.py

```diff
@@ -414,15 +414,15 @@ def run_setup(extensions):
     }
 
     setup(
-        name='cassandra-driver',
+        name='scylla-driver',
         version=__version__,
-        description=' DataStax Driver for Apache Cassandra',
+        description='Scylla Driver for Apache Cassandra',
         long_description=long_description,
-        url='http://github.com/datastax/python-driver',
+        url='http://github.com/scylladb/python-driver',
         project_urls={
-            'Documentation': 'https://docs.datastax.com/en/developer/python-driver/latest/',
-            'Source': 'https://github.com/datastax/python-driver/',
-            'Issues': 'https://datastax-oss.atlassian.net/browse/PYTHON',
+            'Documentation': 'http://scylladb.github.io/python-driver/',
```

Comment:
Consider HTTPS, I think you get that for free from github.
Also, there's nothing at this URL yet 😅

Reply:
now there is something there...